[Build script] Fix debug messages

Devjiu opened the pull request:
Remove debug messages. There are still possible misuse of script as options without "-" in the start will be ignored.

leshikus reviewed (commented):
Not using `-e` is a bad practice; moreover, `-e` should be set at the script start with `set -e`

I don't like removing it

If you have a command `cmd`, which could potentially fail, and you want to handle that, use `if cmd` or `cmd ||`

Devjiu commented:
> Not using `-e` is a bad practice; moreover, `-e` should be set at the script start with `set -e`
> 
> I don't like removing it
> 
> If you have a command `cmd`, which could potentially fail, and you want to handle that, use `if cmd` or `cmd ||`

I've reworked this part based on Andrey's comments, now the options parser is a little simpler and I've removed the bash syntax since it's not needed yet.
